vspace/src/main/webapp/WEB-INF/views/exhibition/module.jsp

```diff
@@ -28,6 +28,15 @@ body {
     function closeLangNav() {
         document.getElementById("myLangSidenav").style.width = "0";
     }
+    $( document ).ready(function() {
+    	if($.browser.chrome) {
+       		alert(1);
+     	} else if ($.browser.mozilla) {
+        	alert(2);
+     	} else if ($.browser.msie) {
+        	alert(3);
```

Comment:
what is this for?

Reply:
It was for debugging purpose, deleted the code.